zygrader/config/preferences.py

```diff
@@ -28,15 +28,22 @@
     "clear_filter": True,
     "vim_mode": False,
     "dark_mode": True,
-    "christmas_mode": False,
-    "spooky_mode": False,
+    "theme": "Default",
+    "unicode_mode": False,
     "browser_diff": False,
     "save_password": False,
     "class_code": "No Override",
     "editor": "Pluma",
     "data_dir": "",
 }
 
+THEMES = {
```

Comment:
These aren't used anywhere in this file, but they are used in `themes.py`. I think it would be more logical to store this data there.